api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -35,9 +35,12 @@
 	private String description;
 	
 	private Set<Integer> memberIds;
-	
+
+	private Set<Integer> members;
```

Comment:
I would change the type to Collection instead of Set but keep a set implementation